Bench/Info.plist

```diff
@@ -17,9 +17,9 @@
 	<key>CFBundlePackageType</key>
 	<string>APPL</string>
 	<key>CFBundleShortVersionString</key>
-	<string>0.29.1</string>
+	<string>0.30.0</string>
```

Comment:
shouldn't the build version string incorporate the beta identifier?

Reply:
I’ve never been completely sure about what we can technically include in the marketing version, but this key is documented to be in x.y.z format.

Reply:
No worries, just checking. As long as we're consistent.